versioned_docs/version-0.5.0/tutorials/kubeslice-cli-demo-on-cloud-clusters.mdx

```diff
@@ -60,16 +60,15 @@ kubeslice-cli --config=<topology-configuration-file> install
 The above command installs the KubeSlice Controller, creates a project, and registers the worker cluster with the project by 
 installing the Slice Operator on the worker cluster.
 
-## Register a Worker Cluster
+## Register a New Worker Cluster
 
-The kubeslice-cli allows you to add a new worker cluster to an existing KubeSlice configuration.
+To register a new worker cluster with the existing KubeSlice configuration (or KubeSlice Controller), follow these steps:
 
-:::note
-Add new worker cluster information to the custom topology file that was used to install KubeSlice earlier and 
-using **install** command, register the new worker cluster with the KubeSlice controller.
-:::
+1. Add new worker cluster information under workers in the custom topology file that was used to install KubeSlice earlier.
```

Comment:
make workers as a property in the template as under `workers`

Reply:
done